### ISSUE TITLE
Attachment cannot be handled correctly, because port number for attachment is missing

### ISSUE THREAD

**Issue by monkeyotg** [wekan/wekan#5281]
## Issue
Attachment cannot be handled correctly, because port number for attachment is missing

### Server Setup Information

* Did you test in newest Wekan?: 7.30.0
* Did you configure root-url correctly so Wekan cards open correctly (see https://github.com/wekan/wekan/wiki/Settings)? Yes
* Operating System: Windows Server 2022
* Deployment Method (Snap/Docker/Sandstorm/bundle/source): Docker

### Problem description
![Snipaste_2024-02-02_01-46-22](https://github.com/wekan/wekan/assets/8792193/c742ee34-d75a-45ae-bd22-23ba410a2841)

**Comment by xet7** [comment]
@monkeyotg 

Do you have port number in this setting?
```
ROOT_URL=http://192.168.178.198:4444
```

**Comment by monkeyotg** [comment]
> @monkeyotg
> 
> Do you have port number in this setting?
> 
> ```
> ROOT_URL=http://192.168.178.198:4444
> ```

Yes, I have set port forward. If manually add the port number, in my case 4444, to the attachment's URL. The attachment (the pic) will be displayed correctly.

**Comment by xet7** [comment]
Try newest WeKan.
